howto/cluster/scale-down.md

```diff
@@ -1,24 +1,35 @@
 Scaling down a LXD cluster involves more checks than scaling up.
 
-First, you must pick a suitable candidate node. When a node is removed, all still running containers on it are stopped and potentially connected users are disconnected. To avoid this, AMS provides a feature to mark a node as unschedulable so that it will not be considered for any further container launches.
+There are two important things to know when scaling down:
+ - The node you remove must not have any container left.
+ - You must wait for a node to be fully removed before you can start removing another one
 
-You can mark a node as unschedulable with the following command:
+## Prepare the node for removal
+
+First, pick the node you want to remove and tell AMS to stop considering this node for new containers:
 
     amc node set lxd0 unscheduable true
 
 (Note that the typo in the command will be fixed in a future Anbox Cloud release.)
 
-Now the node won't be considered for any further container launches. As it might still host containers, you can decide to either kill all containers or wait for your users to disconnect. You can check with the following command if the node still hosts containers:
+Now the node won't be considered for any further container launches.
+
+The node might still have containers running. You can decide to either kill all containers or wait for your users to disconnect.
+Use the following command if the node still hosts containers:
```

Comment:
```suggestion
Use the following command to check whether the node still hosts containers:
```
This is for checking, not for removing - correct?

Reply:
Correct !